public/docs/ts/latest/cookbook/graphql.jade

```diff
@@ -0,0 +1,688 @@
+
+:marked
+  In the constant `schema`, `makeExecutableSchema` has two properties, 
+  `typeDefs` and `resolvers`. Here, you define them with the `typeDefinitions` 
+  and `resolveFunctions` that you created earlier in `in-memory-graphql.ts`. 
+  This way, your GraphQL server knows where to look for definitions and resolvers.
+
+:marked
+  Now that you have an executable schema, execute it using the `graphql` 
+  library and export it so you can use it with the Apollo Client. 
+  First, `npm install`:
+code-example(language="sh" class="code-shell").
+    npm install graphql --save    
+
+.alert.is-important
+  :marked
+    Uri: can you provide the npm install command?
+    :Kapunahele - done
+
+:marked
+  Next, add an import statement for `execute`.
+
++makeExample('heroes-graphql/ts/src/app/in-memory-graphql.ts', 'import-graphql', 'in-memory-graphql.ts (excerpt)')
+:marked
+  Now create a new `networkInterface` class and call it `InBrowserNetworkInterface`.
+
+  That class will have a `schema` property which it will initialize in the constructor.
+
+  Then create a `query` function that will recieve a query as a variable and execute that query using
+  `graphql` execute function against the schema property.
+
+  You send empty objects to the `rootValue` and `contextValue` arguments of the function and send the 
+  `variables` and `operationName` arguments that are related to the query request.
+  
+  Now export the new `InBrowserNetworkInterface` class in order to import it to Apollo Client.
+.alert.is-important
+  :marked
+    Uri: Could you explain what's happening in the below? 
+    This will probably take 2-3 short paragraphs. Just a 
+    general tour from top to bottom.
+    :Kapunahele - let me know if that works
++makeExample('heroes-graphql/ts/src/app/in-memory-graphql.ts', 'execute-and-export', 'in-memory-graphql.ts (excerpt)')
+:marked
+  Now all that's left is to connect the new in-memory server to the Apollo Client configuration 
+  by importing `networkInterface` and adding it to the `client` constant in `client.ts`.
++makeExample('heroes-graphql/ts/src/app/client.ts', '', 'client.ts')
+:marked
+  That's it.  Now you can run your application as if you had a GraphQL server connected to it. 
+  However, there is no persistance&mdash;everything is running in-memory in the browser, 
+  so when you refresh the page, all changes will be lost.
+
+  Now that you have a local server set up, you have some options:
+  * You can store everything on the browser's local-storage using local-storage database libraries.
+  * You can make the resolver functions call your server's existing REST endpoint.
+  * You can start a separate Node GraphQL server and simply move the code into it for persistance.
+
+.alert.is-important
+  :marked
+    Uri: Let's add a conclusion showing what was accomplished in the doc. 
+    Maybe something like (please add anything I've missed):
+    :Kapunahele - done, but not sure if its enough?
```

Comment:
Let's save the details about the conclusion for when we have finalized everything. Then we can make sure it has everything in the same way we are planning for the table of contents.